reference/release-notes/1.22.1.md

```diff
@@ -0,0 +1,38 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
+* Android security updates for June 2024 (see [Android Security Bulletin - June 2024](https://source.android.com/docs/security/bulletin/2024-06-01) for more information).
+* The Android WebView has been updated to [125.0.6422.147](https://chromereleases.googleblog.com/2024/05/chrome-for-android-update_30.html).
+
+### Bug fixes
+
+The following bugs are fixed in this release:
+
+*  [LP 2066338](https://bugs.launchpad.net/anbox-cloud/+bug/2066338) Failed to join a session with the following error:
+
+    ```
+    Service: Failed to generate new generation for session cp5oh64ut74s9inbvt3g: database is locked
+    ```
+* [LP 2067735](https://bugs.launchpad.net/anbox-cloud/+bug/2067735) AMS couldn't create instances with an error message similar to the following (occurs sporadically):
+
+    ```
+    Failed creating instance from image: Could not locate a zvol for ams0/containers/ams-cpc64323ho619qhebfrg
+    ```
+* [LP 2067455](https://bugs.launchpad.net/anbox-cloud/+bug/2067455) Juju controller is not selected during appliance upgrade and the upgrade fails.
+* [LP 2064923](https://bugs.launchpad.net/anbox-cloud/+bug/2064923) `nvidia-smi` command is not detected correctly and rendering falls back to EmuGL.
+* [LP 2065401](https://bugs.launchpad.net/anbox-cloud/+bug/2065401) `amc list` always returns a full instance list when filtering with `app=nonexistent`.
+
+## Known issues
+<!-- wokeignore:rule=master -->
+* [LP 2069021](https://bugs.launchpad.net/anbox-cloud/+bug/2069021) Recent changes to the [AMS SDK](https://github.com/canonical/ams-sdk) has caused the examples provided in the [README](https://github.com/canonical/ams-sdk/blob/master/README.md) and other [examples provided in the repository](https://github.com/canonical/ams-sdk/tree/master/examples/ams) to stop working. This will be fixed in the 1.23.0 release.
```

Comment:
... has caused the examples provided in the README and other examples provided in the repository ...
... has caused the golang-based projects developed and built with AMS SDK to stop working. This will be fixed in the 1.23.0 release.